packages/next/src/export/index.ts

```diff
@@ -197,6 +197,15 @@ export default async function exportApp(
       )
     }
 
+    // Running next export or using output: 'export' in next.config.js
+    if (!options.buildExport || nextConfig.output === 'export') {
+      if (nextConfig.experimental.serverActions) {
+        throw new ExportError(
```

Comment:
Can we put this in line 183 and 184 with the others?

Reply:
Can't fully merge as the 2nd condition represents wider ranger than the previous one. Refactored a bit to make it more readable